clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionServiceTest.java

```diff
@@ -1,16 +1,8 @@
 package com.linkedin.davinci.kafka.consumer;
 
-import static com.linkedin.venice.ConfigKeys.KAFKA_BOOTSTRAP_SERVERS;
-import static com.linkedin.venice.ConfigKeys.KAFKA_ZK_ADDRESS;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static com.linkedin.venice.ConfigKeys.*;
```

Comment:
Let's not revert that. There is an IDE setting for avoiding this. Please check with @anaberezhnov for details.

Reply:
Sure, will check

Reply:
Here it is: https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports

Reply:
I think I resolved this, could you take a second look? Thanks!